commands-yml/commands/mobile-command.yml

```diff
@@ -84,6 +84,7 @@
       | mobile:navigateTo | Action to [NavigationView](https://developer.android.com/reference/android/support/design/widget/NavigationView) by [NavigationViewActions#navigateTo](https://developer.android.com/reference/android/support/test/espresso/contrib/NavigationViewActions.html#navigateTo(int)). The view must be a child of a DrawerLayout, of type NavigationView, visible on screen and displayed on screen. | `{element, menuItemId}` | `{element: elementId, menuItemId: 1}` |
       | mobile:scrollToPage | Action to [ViewPager](https://developer.android.com/reference/android/support/v4/view/ViewPager) by [ViewPagerActions](https://developer.android.com/reference/android/support/test/espresso/contrib/ViewPagerActions). `scrollTo` must be one of `first`, `last`, `left`, `right`. `scrollTo` is used by default if `scrollTo` and `scrollToPage` are provided. | `{scrollTo, scrollToPage, smoothScroll}` | `{element: elementId, scrollTo: 'left', smoothScroll: true}`, `{element: elementId, scrollToPage: 2}`|
       | mobile:backdoor | Invoke arbitrary methods defined in Android app. The methods must be `public`. `target` is `activity` or `application`. `methods` are methods you would like to invoke. Read docstring of _mobileBackdoor_ method [here](https://github.com/appium/appium-espresso-driver/blob/master/lib/commands/general.js) for more details | `{target, methods}` | `{target: 'activity', methods: [{name: 'method1'}, {name: 'method2', args: [{value: 'hello appium', type: 'java.lang.String'}] }] }` |
+      | mobile:flashElement | Flashes an element on the screen to visually confirm you are operating on the correct element | `{elementId}` | `{elementId: "1234-abcd-5678}` |
```

Comment:
Should we mention about `{durationMillis, repeatCount}` as well? (they are optional.)
https://github.com/appium/appium-espresso-driver/pull/337/files#diff-b6b44c1feede68b04fcec6008c7e9e2dR14
https://github.com/appium/appium-espresso-driver/pull/337/files#diff-046460f6a24bb8c575cc6d5f55189736R191